cms/static_src/sass/main.scss

```diff
@@ -3,24 +3,14 @@ Don't add css code directly to this file, add it to the relevant component file
 
 // New components not yet in the design system
 @use 'components/accreditations';
-@use 'components/article-header';
-@use 'components/button-nav';
 @use 'components/census-logo';
-@use 'components/common-header';
 @use 'components/corrections-and-notices';
-@use 'components/document-list-block';
 @use 'components/chart-embed';
 @use 'components/featured-article';
-@use 'components/panel';
-@use 'components/release-header';
 @use 'components/rich-text';
 @use 'components/spacing';
-@use 'components/topic-header';
 
 // ONS design system overrides (until these are no longer needed)
 @use 'components/design_system_overrides/back-to-top';
-@use 'components/design_system_overrides/button';
-@use 'components/design_system_overrides/container';
 @use 'components/design_system_overrides/grid';
 @use 'components/design_system_overrides/header';
```

Comment:
question: we now use the corrections/notice, header, accreditations components from the DS. Can the additional styling be removed? is it subject for another PR?

Reply:
No, let's keep them incremental. Was asking to check whether it is on your radar